lib/rules/indent.js

```diff
@@ -850,7 +850,18 @@ module.exports = {
 
             ExportNamedDeclaration(node) {
                 if (node.declaration === null) {
-                    addElementListIndent(getTokensAndComments(node).slice(1), node.specifiers, 1);
+                    const tokensInNode = getTokensAndComments(node);
+                    const closingCurly = sourceCode.getLastToken(node, astUtils.isClosingBraceToken);
+                    const closingCurlyIndex = lodash.sortedIndexBy(tokensInNode, closingCurly, token => token.range[0]);
+
+                    // Indent the specifiers in `export {foo, bar, baz}`
+                    addElementListIndent(tokensInNode.slice(1, closingCurlyIndex + 1), node.specifiers, 1);
+
+                    if (node.source) {
+
+                        // Indent everything after the `from` token in `export {foo, bar, baz} from 'qux'`
```

Comment:
Is this indenting everything after `from`, or everything after and including `from`? If the latter, could the comment be updated for clarity?